omod/src/main/resources/apps/refapp_registrationapp_app.json

```diff
@@ -5,6 +5,16 @@
         "label": "Register Patient",
         "description": "Create a new Patient Record",
         "extensions": [
+            {
+                "id": "application.registrationapp.summary.editPatientLink",
+                "extensionPointId": "patientDashboard.overallActions",
+                "type": "link",
+                "label": "Summary/ Edit Patient",
```

Comment:
This should be either of
* >`"Edit Registration Information"`
* >`"Edit Registration"`
* >`"Registration Summary"`

... and should be provided as an i18n message.

Reply:
@mogoodrich any preference, or other suggestions?

Reply:
@Ruhanga can you change this first:
```json
"label": "Edit Registration Information"
```

@dkayiwa after that we're good to merge I think ;-)

Reply:
@Ruhanga also can you rely on a i18n message for this label, if possible?

Reply:
I have worked on the comments @mks-d, except I'm thinking i18n should have been done with in the Reg App. This is trivial though.